frontend/package.json

```diff
@@ -22,6 +22,7 @@
     "@types/react-dom": "^17.0.0",
     "axios": "^0.21.1",
     "date-fns": "2.11.1",
+    "firebase": "^9.6.8",
```

Comment:
Nothing is broken by specifying the firebase version, right?

Reply:
I don't think so. `yarn install` for firebase automatically put this value in the package.json.